styles.css

```diff
@@ -57,8 +67,501 @@ ul a {
   color: #fff;
 }
 
+#main-nav {
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  box-shadow: 0 0 32px rgba(0, 0, 0, 0.15);
+  font-size: 1.3em;
+  font-family: "Ropa Sans", sans-serif;
+}
+
+#nav-logo {
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  height: 100%;
+  width: 100px;
+}
+
+#nav-logo img {
+  margin: 15px;
+}
+
+#main-nav ul {
+  padding: 5px 0;
+}
+
+#header-icons {
+  justify-content: space-around;
+}
+
+#header-icons i {
+  border-left: 1px solid #eee;
+  padding: 15px;
+}
+
+#main-nav ul li {
+  margin-right: 35px;
+  list-style-type: none;
+  padding: 10px 0;
+}
+
+#main-nav a,
+#header-icons i {
+  color: #aaa;
```

Comment:
### Using the same Rule repetitively. 

Avoid repeat same rule `color: #d33a2c;` tons of time. Use  instead `.gray{color: #d33a2c;}`

Reply:
```
color: #aaa;